src/components/color-system/examples/colors-in-components.tsx

```diff
@@ -0,0 +1,41 @@
+import { Component, h } from '@stencil/core';
+
+@Component({
+    tag: 'limel-example-colors-in-components',
+    shadow: true,
+    styleUrl: 'colors-in-components.scss',
+})
+export class ColorExample {
+    public render() {
+        console.log('----------------------')
```

Comment:
I assume this console.log is just a left-over from trying stuff out? If so, I'll remove it.

Reply:
I've removed this.